### ISSUE TITLE
bug(YouTube/Hide feed components): `Hide mix playlists` setting hides components in channel profile

### ISSUE THREAD

**Issue by Aryan-Chauhan** [inotia00/ReVanced_Extended#2024]
### Type

Cosmetic

### Tools used

Revancify

### Application

YT v19.16.39

### Bug description

`Hide Mix Playlist` is hiding unintended UI elements.

When it is enabled:
![Screenshot_2024-05-15-19-35-11-79_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/a8d871d3-6d84-42ee-aa72-18c97fad5487)
![Screenshot_2024-05-15-19-35-57-90_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/8534363e-8c86-4c87-83e7-d5485a64267b)
![Screenshot_2024-05-15-19-36-09-89_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/b9a2bdd0-0f71-4111-b96b-aa0513fff7cd)

When it is disabled:
![Screenshot_2024-05-15-19-33-25-75_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/a4e76161-87a2-4dc3-a586-080166214690)
![Screenshot_2024-05-15-19-33-54-72_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/c2b098e8-e2e1-4e38-abf6-9b769608e861)
![Screenshot_2024-05-15-19-34-03-71_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/540cdc00-b716-4155-a98b-efa83fdd76dd)

### Error logs

```shell
No logs.
```

### Solution

`Hide Mix Playlist` shouldn't hide unintended UI elements.

### Additional context

Channel links:

https://youtube.com/@watchmojo
https://youtube.com/@unveiled
https://youtube.com/@mojotravels

![Screenshot_2024-05-15-19-44-47-85_a9acfd2263b6b7a65bf2540b0d9449ca](https://github.com/inotia00/ReVanced_Extended/assets/91815107/ed7e8219-7a5e-41a3-a08f-797daa0c857d)

### Device Environment

Realme 7 Pro, Android 12 (Realme UI 3.0)

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
fixed in [revanced-patches-4.8.0-dev.6](https://github.com/inotia00/revanced-patches/releases/tag/v4.8.0-dev.6)

**Comment by Aryan-Chauhan** [comment]
Thanks!

Why the "Partially resolved" label?

Are there any side-affects?

**Comment by inotia00** [comment]
Currently only released in the `dev` branch
This issue will be closed once everything is merged into the `main` branch

**Comment by inotia00** [comment]
fixed in [revanced-patches-4.8.3](https://github.com/inotia00/revanced-patches/releases/tag/v4.8.3)
